finagle-http/src/main/scala/com/twitter/finagle/http/filter/StatsFilter.scala

```diff
@@ -1,9 +1,29 @@
 package com.twitter.finagle.http.filter
 
-import com.twitter.finagle.{Service, SimpleFilter}
-import com.twitter.finagle.stats.StatsReceiver
+import com.twitter.finagle._
 import com.twitter.finagle.http.{Request, Response, Status}
-import com.twitter.util.{Duration, Future, Return, Stopwatch, Throw}
+import com.twitter.finagle.stats.StatsReceiver
+import com.twitter.util._
+
+object StatsFilter {
+  val role = Stack.Role("HttpStatsFilter")
+
+  def module[Req <: Request]: Stackable[ServiceFactory[Req, Response]] =
```

Comment:
I honestly don't think we still need this `Req <: Request` type-param. *I think* we've done this before to support something we called "rich http" (which is now just `finagle-http`). Please, correct me if I'm wrong. /cc @mosesn @dschobel @roanta